packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -119,6 +114,30 @@ const VirtualListBaseFactory = (type) => {
 			 */
 			initUiChildRef: PropTypes.func,
 
+			/**
+			 * The Function that returns `true` if the item at the index is disabled
+			 *
+			 * Usage:
+			 * ```
+			 * isDisabledItem = (index) => (this.items[index].disabled)
```

Comment:
Would `isItemDisabled` make more sense?  It reads more as a sentence than the other name.